<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Clock</class>
 <widget class="QWidget" name="Clock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clock</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(14, 19, 22);
color: rgb(255, 255, 255);
border-radius:12px;
opacity:0.95;
border-radius:6px 6px 0px 0px;</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>660</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(102, 102, 102);
</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>454</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="count_stat">
     <property name="geometry">
      <rect>
       <x>178</x>
       <y>476</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(37, 200, 124);
font: 11pt &quot;Sans Serif&quot;;

</string>
     </property>
     <property name="text">
      <string>开始</string>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>454</width>
       <height>461</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QWidget" name="page_4">
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>129</x>
         <y>178</y>
         <width>195</width>
         <height>45</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 36pt &quot;Sans Serif&quot;;</string>
       </property>
       <property name="text">
        <string>00:00:00</string>
       </property>
      </widget>
      <widget class="QPushButton" name="min_5btn">
       <property name="geometry">
        <rect>
         <x>24</x>
         <y>404</y>
         <width>56</width>
         <height>36</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">width:56px;
height:36px;
background:rgba(46,46,46,1);
border-radius:4px;
</string>
       </property>
       <property name="text">
        <string>5分钟</string>
       </property>
      </widget>
      <widget class="QPushButton" name="min_10btn">
       <property name="geometry">
        <rect>
         <x>111</x>
         <y>404</y>
         <width>56</width>
         <height>36</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">width:56px;
height:36px;
background:rgba(46,46,46,1);
border-radius:4px;
</string>
       </property>
       <property name="text">
        <string>10分钟</string>
       </property>
      </widget>
      <widget class="QPushButton" name="min_20btn">
       <property name="geometry">
        <rect>
         <x>199</x>
         <y>404</y>
         <width>56</width>
         <height>36</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">width:56px;
height:36px;
background:rgba(46,46,46,1);
border-radius:4px;
</string>
       </property>
       <property name="text">
        <string>20分钟</string>
       </property>
      </widget>
      <widget class="QPushButton" name="min_30btn">
       <property name="geometry">
        <rect>
         <x>287</x>
         <y>404</y>
         <width>56</width>
         <height>36</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">width:56px;
height:36px;
background:rgba(46,46,46,1);
border-radius:4px;
</string>
       </property>
       <property name="text">
        <string>30分钟</string>
       </property>
      </widget>
      <widget class="QPushButton" name="min_60btn">
       <property name="geometry">
        <rect>
         <x>374</x>
         <y>404</y>
         <width>56</width>
         <height>36</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">width:56px;
height:36px;
background:rgba(46,46,46,1);
border-radius:4px;
</string>
       </property>
       <property name="text">
        <string>60分钟</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_5">
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>129</x>
         <y>178</y>
         <width>195</width>
         <height>45</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 36pt &quot;Sans Serif&quot;;</string>
       </property>
       <property name="text">
        <string>00:00:00</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>87</x>
         <y>405</y>
         <width>280</width>
         <height>36</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">width:280px;

height:36px;
background:rgba(72,72,76,1);
border-radius:4px;
</string>
       </property>
       <item>
        <property name="text">
         <string>默认铃声</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>...</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>178</x>
         <y>137</y>
         <width>20</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/icon-4-16x16.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>140</y>
         <width>70</width>
         <height>14</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">width:70px;
height:14px;
font-size:14px;
font-family:Source Han Sans CN;
font-weight:400;
color:rgba(255,255,255,1);
opacity:0.45;
</string>
       </property>
       <property name="text">
        <string>下午05:31</string>
       </property>
      </widget>
      <widget class="QPushButton" name="count_push">
       <property name="geometry">
        <rect>
         <x>193</x>
         <y>259</y>
         <width>68</width>
         <height>68</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>128</x>
       <y>42</y>
       <width>197</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 36pt &quot;Sans Serif&quot;;
</string>
     </property>
     <property name="text">
      <string>00:00:00</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addAlarmBtn">
     <property name="geometry">
      <rect>
       <x>177</x>
       <y>476</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(37, 200, 124);
font: 11pt &quot;Sans Serif&quot;;



</string>
     </property>
     <property name="text">
      <string>添加闹钟</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>142</x>
       <y>110</y>
       <width>173</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
color: rgb(148, 148, 148);
font: 12pt &quot;Sans Serif&quot;;
opacity:0.45;</string>
     </property>
     <property name="text">
      <string>点击闹钟显示剩余时间</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>39</x>
       <y>140</y>
       <width>376</width>
       <height>280</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">selection-background-color: rgb(162, 162, 162);
border-radius:0px 0px 6px 6px;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QPushButton" name="pushButton_Start">
     <property name="geometry">
      <rect>
       <x>192</x>
       <y>385</y>
       <width>68</width>
       <height>68</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
color: rgb(148, 148, 148);
font: 12pt &quot;Sans Serif&quot;;</string>
     </property>
     <property name="text">
      <string>00:00:00</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>128</x>
       <y>42</y>
       <width>197</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
text-decoration: underline;
font: 36pt &quot;Sans Serif&quot;;</string>
     </property>
     <property name="text">
      <string>00:00:00</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_timeselect">
     <property name="geometry">
      <rect>
       <x>292</x>
       <y>476</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">width:100px;
height:32px;
background:rgba(44,44,46,1);
border:1px solid rgba(68,68,71,1);
border-radius:4px;
</string>
     </property>
     <property name="text">
      <string>复位</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ring">
     <property name="geometry">
      <rect>
       <x>62</x>
       <y>476</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">width:100px;
height:32px;
background:rgba(44,44,46,1);
border:1px solid rgba(68,68,71,1);
border-radius:4px;
</string>
     </property>
     <property name="text">
      <string>计次</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_2">
     <property name="geometry">
      <rect>
       <x>39</x>
       <y>130</y>
       <width>376</width>
       <height>251</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">selection-background-color: rgb(162, 162, 162);
border-radius:0px 0px 6px 6px;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>437</x>
     <y>8</y>
     <width>9</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>8</y>
     <width>10</width>
     <height>9</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>454</width>
     <height>633</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="page_6">
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>62</x>
       <y>557</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">width:100px;
height:32px;
background:rgba(44,44,46,1);
border:1px solid rgba(68,68,71,1);
border-radius:4p</string>
     </property>
     <property name="text">
      <string>删除闹铃</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>292</x>
       <y>557</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">width:100px;
height:32px;
background: rgb(37, 200, 124);
border:1px solid rgba(68,68,71,1);
border-radius:4p</string>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>454</width>
     <height>633</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="page_7">
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>332</x>
       <y>8</y>
       <width>60</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>43</y>
       <width>42</width>
       <height>15</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 10pt &quot;Sans Serif&quot;;</string>
     </property>
     <property name="text">
      <string>倒计时</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>43</y>
       <width>42</width>
       <height>15</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 10pt &quot;Sans Serif&quot;;</string>
     </property>
     <property name="text">
      <string>闹钟</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>43</y>
       <width>42</width>
       <height>15</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 10pt &quot;Sans Serif&quot;;</string>
     </property>
     <property name="text">
      <string>秒表</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>63</x>
       <y>8</y>
       <width>60</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>197</x>
       <y>8</y>
       <width>60</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>660</width>
       <height>1</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(102, 102, 102);
</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionOn">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/Images/Images/NO.png</normaloff>:/Images/Images/NO.png</iconset>
   </property>
   <property name="text">
    <string>On</string>
   </property>
  </action>
  <zorder>stackedWidget_2</zorder>
  <zorder>line</zorder>
  <zorder>pushButton_5</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>stackedWidget_3</zorder>
  <zorder>stackedWidget</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
